pkgs/desktops/mate/mate-themes/default.nix

```diff
@@ -16,6 +16,10 @@ stdenv.mkDerivation rec {
 
   propagatedUserEnvPkgs = [ gtk-engine-murrine ];
 
+  postInstall = ''
+    "${gtk3.out}/bin/gtk-update-icon-cache" "$out"/share/icons/ContrastHigh
```

Comment:
To support cross compilation you should put gtk3 in nativeBuildInputs and just use gtk-update-icon-cache

Reply:
Done.